.buildkite/verify-yaml.sh

```diff
@@ -3,9 +3,9 @@
 set -ex
 cd "$(dirname "${BASH_SOURCE[0]}")/.."
 
-gcloud container clusters get-credentials dogfood --zone us-central1-a --project sourcegraph-dev
+gcloud container clusters get-credentials dogfood --zone us-central1-f --project sourcegraph-dogfood
 
-kubectl apply --dry-run --validate --recursive -f base/ --context=gke_sourcegraph-dev_us-central1-a_dogfood
-kubectl apply --dry-run --validate --recursive -f configure/ --context=gke_sourcegraph-dev_us-central1-a_dogfood
+kubectl apply --dry-run --validate --recursive -f base/ --context=gke_sourcegraph-dogfood_us-central1-f_dogfood
+kubectl apply --dry-run --validate --recursive -f configure/ --context=gke_sourcegraph-dogfood_us-central1-f_dogfood
```

Comment:
I believe `--dry-run=server` is what we want here. Getting logs about `--dry-run` being deprecated. Opened PR here https://github.com/sourcegraph/deploy-sourcegraph-dot-com/pull/3526.